mercury.spec

```diff
@@ -107,6 +108,9 @@ cd build
 
 
 %changelog
+* Wed Nov 10 2021 Alexander Oganezov <alexander.a.oganezov@intel.com> - 2.1.0~rc3-1
+- Update to version v2.1.0rc3
```

Comment:
Add a line documenting the addition of the patch?

Reply:
will add

Reply:
After internal discussion between ofi/cart/mercury, it was decided to not include this patch